README.md

```diff
@@ -120,6 +139,29 @@ Encoders configurations
 
 |Key| Value|Desc |Default|
 |---|---|---|---|
+|sbc_min_bp|2-250|minimum allowed bitpool|auto|
+|sbc_max_bp|2-250|maximum allowed bitpool, may not be < sbc_min_bp|auto|
+|sbc_freq|8|16000 Hz sample frequency|0|
+||4|32000 Hz sample frequency|
+||2|44100 Hz sample frequency|
+||1|48000 Hz sample frequency|
+||0|do not enforce sample frequency (default)|
+|sbc_cmode|8|mono channel-mode|0|
+||4|dual channel-mode|
+||2|stereo channel-mode|
+||1|joint stereo channel-mode|
+||0|do not enforce channel-mode (default)|
+|sbc_alloc|2|use SNR bit-allocation algorithm|0|
+||1|use loudness bit-allocation algorithm|
+||0|do not enforce bit-allocation algorithm (default)|
+|sbc_sbands|2|4 subbands|0|
+||1|8 subbands|
+||0|do not enforce subbands count (default)|
+|sbc_blen|8|4 audio blocks in one audio frame|0|
+||4|8 audio blocks|
+||2|12 audio blocks|
+||1|16 audio blocks|
+||0|do not enforce audio blocks count (default)|
```

Comment:
It would be better to use readable value;
For example, `16k` for `16000 Hz sample frequency`, `mono` for `mono channel-mode`, and use `auto` for default behavior.